dash-pipeline/bmv2/stages/trusted_vni.p4

```diff
@@ -0,0 +1,48 @@
+#ifndef _DASH_STAGE_TRUSTED_VNI_P4_
+#define _DASH_STAGE_TRUSTED_VNI_P4_
+
+control trusted_vni_stage(
+    inout headers_t hdr,
+    inout metadata_t meta)
+{
+    action permit() {}
+
+    action deny() {
+        meta.dropped = true;
+    }
+
+    @SaiTable[single_match_priority = "true", api = "dash_trusted_vni", order=1, isobject="false"]
```

Comment:
will be better to split the files, so we can have both batch API working.

Reply:
The one file is better organized for same category of trusted vni objects, and their batch APIs are working in orchagent.